src/main/sensors/acceleration_init.c

```diff
@@ -93,7 +93,8 @@
     !defined(USE_ACCGYRO_BMI270) && !defined(USE_ACC_SPI_ICM42605) && !defined(USE_ACC_SPI_ICM42688P) && \
     !defined(USE_ACC_ADXL345) && !defined(USE_ACC_BMA280) && !defined(USE_ACC_LSM303DLHC) && \
     !defined(USE_ACC_MMA8452) && !defined(USE_ACC_LSM303DLHC) && !defined(USE_ACCGYRO_LSM6DSV16X) && \
-	!defined(USE_VIRTUAL_ACC)
+    !defined(USE_VIRTUAL_ACC) \
+    && !defined(USE_ACC_SPI_ICM20649)
```

Comment:
prefer `virtual` last, but only an opnion.

Reply:
Agree, lazy addition, as we do not use that gyro in any config, will update.

Reply:
Reordered.